modules/openxr/extensions/openxr_fb_passthrough_extension_wrapper.h

```diff
@@ -234,12 +233,14 @@ class OpenXRFbPassthroughExtensionWrapper : public OpenXRExtensionWrapper, publi
 		XR_PASSTHROUGH_IS_RUNNING_AT_CREATION_BIT_FB,
 		XR_PASSTHROUGH_LAYER_PURPOSE_RECONSTRUCTION_FB,
 	};
+	/* TODO Commenting out because unused and CI fails, need to make this work properly.
```

Comment:
@BastiaanOlij This structure can be used to set a specific style for the passthrough layer (e.g. specify its opacity and edge rendering color) through a call to [xrPassthroughLayerSetStyleFB](https://registry.khronos.org/OpenXR/specs/1.0/man/html/xrPassthroughLayerSetStyleFB.html). As this feature is not yet implemented in the extension wrapper, I think it might also be okay to just delete this structure from here for now.

Reply:
Agree! Let's remove it for now until we decide to fully implement it.